lint-staged.config.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @type {{ [key: string]: import("lint-staged").Commands | import("lint-staged").ConfigFn }}
+ */
+const config = {
+  '*.ts': 'eslint --fix',
+  '*.js': 'eslint --fix',
+  '*.json': 'npx prettier --write',
+  '*.yml': 'npx prettier --write'
```

Comment:
We have considered using `prettier`, but it is still in draft. If we decide to use it, we will need to update `eslint-config-hexo` for uniformalization.

 - https://github.com/hexojs/eslint-config-hexo/pull/25
 - https://github.com/hexojs/hexo/pull/3719

Reply:
> Would you please separate the pull request into two parts? type fixes and the others (such as prettier and husky).
> 
> To be honest, there is a possibility that we cannot accept the installation of `prettier` and `husky`.

okay ill separate the PR

Reply:
already separated. please check https://github.com/hexojs/hexo-front-matter/pull/71#issuecomment-1546599676